src/features/activities/activity-form/ActivityEditForm.module.scss

```diff
@@ -2,25 +2,78 @@
 
 .form {
   @include flex(column);
-
   min-height: 100%;
   position: relative;
+  flex-wrap: nowrap;
 
   .inputs {
-    @include flex(column);
-
+    display: flex;
+    flex-direction: column;
     padding: var(--spacing-lg);
     gap: var(--spacing-md);
-    flex-grow: 1;
-    overflow-y: scroll;
+  
+    .categorySelectWrapper {
+      display: flex;
+      flex-direction: column;
+      gap: var(--spacing-md);
+      margin-top: var(--spacing-sm); /* Space between categories and other elements */
+      height: auto; /* Allow height to adjust based on content */
+      min-width: 100%; /* Ensure the wrapper has a minimum width */
+
+      .categorySelect {
+        display: flex;
+        align-items: center;
+        gap: var(--spacing-sm);
+        flex-wrap: nowrap; /* Prevent wrapping */
+        overflow-x: auto; /* Allow horizontal scrolling if needed */
+        
+        .selectedCategoriesWrapper {
+          display: flex;
+          flex-wrap: nowrap;
+          overflow-x: auto; /* Allow horizontal scrolling */
+          gap: var(--spacing-sm);
+        }
+        
+        .label {
+          margin-right: var(--spacing-sm);
+        }
+        
+        .select {
+          display: flex;
+          align-items: center;
+          flex: 1; /* Allow the select to take remaining space */
+          cursor: pointer;
 
-    .titleAndIcon {
-      position: relative;
+          .selectIcon {
+            margin-right: var(--spacing-sm);
+          }
 
-      .iconPicker {
-        position: absolute;
-        right: 0;
+          .placeholder {
+            flex: 1; /* Allow placeholder to take remaining space */
+            color: var(--gray);
+          }
+
+          .toggleSelect {
+            margin-left: auto;
+          }
+        }
```

Comment:
It seems like none of these classes used directly in `ActivityEditForm` but rather refer to the contents of `CategorySelect`. Please move them to `CategorySelect.module.scss`.